scripts/flush.py

```diff
@@ -41,15 +41,14 @@ def main(collection_ex_id):
 
     samples = [sample['id'] for sample in sample_units]
 
+    sample_return = requests.get(case_url + "sampleunitids?sampleUnitId=" + ','.join(samples), auth=config["CASE_AUTH"])
```

Comment:
Getting the following error here when running against a 500 row sample: 

```python
requests.exceptions.HTTPError: 414 Client Error: URI Too Long for url: http://localhost:8171/cases/sampleunitids?sampleUnitId=bde74017-fbef-4851-a0cb...[snip]
```

Reply:
(created using `make setup`)

Reply:
The requests to the casesvc/sampleunitid endpoint are now being sent in chunks of 10.